src/language-js/parse/babel.js

```diff
@@ -121,12 +123,21 @@ function createParse(parseMethod, ...optionsCombinations) {
       );
     }
 
+    const shouldEnableV8intrinsicPlugin = /%[A-Z]/.test(text);
     if (text.includes("|>")) {
-      combinations = pipelineOperatorPlugins.flatMap((pipelineOperatorPlugin) =>
+      const conflictsPlugins = [...pipelineOperatorPlugins];
+      if (shouldEnableV8intrinsicPlugin) {
+        conflictsPlugins.push(v8intrinsicPlugin);
+      }
```

Comment:
```suggestion
      const conflictsPlugins = shouldEnableV8intrinsicPlugin
        ? [...pipelineOperatorPlugins, v8intrinsicPlugin]
        : pipelineOperatorPlugins;
```

If we don't need add  `v8intrinsicPlugin`, there is no need copy that array.